### PR TITLE
[+] Command Generate - Add "Application host" parameter (`-H, --application-host`) to `lumber generate`

### PR DESCRIPTION
**Pull request by rap2hpoutre**
~~Waiting for #269 to be reviewed.~~

### CODE REVIEW

**Comment by rap2hpoutre** [comment]
[Lumber - add --application-host param](https://trello.com/c/9HUlOHSY/3741-lumber-add-application-host-param)

File: lumber-generate.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const logger = require('./services/logger');
 program
   .description('Generate a backend application with an ORM/ODM configured.')
   .option('-c, --connection-url <connectionUrl>', 'Enter the database credentials with a connection URL')
+  .option('-H, --application-host <applicationHost>', 'Hostname of your admin backend app')
```
Comment (arnaudbesnier) [inline lumber-generate.js]: [Design] According to your epic specification `-H` should be `-h`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 ### Added
+- Command Generate - Add  "Application host" parameter (`-p, --application-host`) to `lumber generate`.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: [Conventions] It is a very small detail, but we tend to accumulate changes from top to bottom. So I would move this change at the end of the `### Added` section.

File: lumber-generate.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ const logger = require('./services/logger');
...
   .option('-p, --application-port <applicationPort>', 'Port of your admin backend app')
```
Comment (arnaudbesnier) [inline lumber-generate.js]: [Wording] Also a detail but I would replace "app" by "application" in the option descriptions.
We tend not to "truncate" our variable names, so I thing we should do the same for the wording.
_Even though I agree that "app" is almost a real word now :)_

File: CHANGELOG.md (hunk 2 above)
Comment (arnaudbesnier) [reply CHANGELOG.md]: `-p, --application-host` ? 🤔
With a strong "auto review" you would have detected this, before submitting this PR to your reviewer.

File: lumber-generate.js (hunk 1 above)
Comment (rap2hpoutre) [reply lumber-generate.js]: `-h` is already in use (see `--help`) in all lumber commands, thanks to `command`:

```
➜  ~ lumber generate -h
Usage: lumber-generate [options]

Generate a backend application with an ORM/ODM configured.

Options:
  -c, --connection-url <connectionUrl>  Enter the database credentials with a connection URL
  --no-db                               Use Lumber without a database.
  -h, --help                            output usage information
```

One character parameters are inconsistent by nature, I thought `H` was a good candidate. It is "host" first character. Feel free to suggest another shorthand.

Comment (arnaudbesnier) [reply lumber-generate.js]: Ha Sorry, did think about this constraint. You're god damn right!

**Pull request merged**